tools/dev/config.yaml

```diff
@@ -33,14 +33,25 @@ environments:
     name: janusgraph
     database_config:
       url: http://localhost:8182
+      analytics_engine:
+        enabled: true
+        type: spark-analytics-api
+        url: http://localhost:8100
+  analytics_engine:
+    enabled: true
+    defaultUseAnalyticsEngine: false
   configuration_storage:
     type: etcd
     url: http://localhost:2379
   contextionary:
-    knn_file: contextionary/contextionary.knn
-    idx_file: contextionary/contextionary.idx
+    knn_file: test/contextionary/example.knn
+    idx_file: test/contextionary/example.idx
```

Comment:
This is on the config `janusgraph_test`, which runs in Docker, and in Docker the contextionary is not in a `test` folder?

Reply:
Update: This is not the config that is used in tests, the tests use `janusgraph_docker`. In this case I have absolutely no idea when this config is ever used? Do we even need it?

By the way, this confusion is exactly what 12-factor tries to avoid and why I want to clean this up in #800 :)

Reply:
> Generally looks good, but I don't understand the changes on the config environment `janusgraph_test` of which I also don't know if it's ever used or not?

Hi @etiennedi I'm using two types of dev setups. One with, and one without the network. The janus_test setup is used for that, see: https://github.com/creativesoftwarefdn/weaviate/blob/feature/version_and_noGQL/tools/dev/run_dev_server_no_network.sh#L14
Now, I know this refs the JSON file, but I wasn't sure if you want to move to yaml 100% or not.